mypy/config_parser.py

```diff
@@ -1,15 +1,14 @@
 import argparse
-from collections import OrderedDict
```

Comment:
Based on the discussion in https://github.com/python/mypy/pull/10219 it seemed like OrderedDict was only needed by Python 3.5 which is no longer supported, so guessing we don't need it anymore?